src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,256 @@
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use selection boxes for choices made during onboarding or other one-time processes."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-usage-1-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline
+      >
+        <p>
+            Use selection boxes when the available choices are abstract or potentially unfamiliar to the user.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use selection boxes to represent choices or categories that are new to the user."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-usage-2-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don’t use selection boxes for repeatable data entry tasks, especially inside modals. Use other selection controls, like checkboxes or radio buttons.
+        </p>
+        <p>
+          Don't use selection boxes when there are more than 12 options. Use other selection controls instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use selection boxes when the choice is part of a task or workflow that users will complete repeatedly."
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-usage-3-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use other selection inputs for choices that are part of task workflows."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/selection-box/selection-box-usage-4-modern.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+    </sky-docs-demo-page-section>
+    
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/selection-box/selection-box-anatomy-modern.png"
+      >
+        <sky-docs-anatomy-item>Box</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Checkbox or radio button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Title</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Icon</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Description</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
```

Comment:
Would it make sense to say that if you are using neither an icon nor a description that you should use a regular checkbox/radio? I.e. individually they are optional, but you must use at least one of them.

Reply:
If you rephrase it in line with how we describe when to use icons/descriptions, I think it would make sense.

"If neither the icon nor the description provide additional meaning, why did you even think you needed this in the first place?"